java/ql/src/change-notes/2023-03-08-insecure-ldap-authentication.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* The query `java/insecure-ldap-auth` has been promoted from experimental to the main query pack. This query was originally [submitted as an experimental query by @luchua-bc](https://github.com/github/codeql/pull/4854)
```

Comment:
```suggestion
* The query `java/insecure-ldap-auth` has been promoted from experimental to the main query pack. This query detects transmission of cleartext credentials in LDAP authentication. Insecure LDAP authentication causes sensitive information to be vulnerable to remote attackers. This query was originally [submitted as an experimental query by @luchua-bc](https://github.com/github/codeql/pull/4854)
```
Suggesting adding a little more information about the purpose of this query to the changelog! I was unsure if the third sentence, "`Insecure LDAP authentication causes sensitive information to be vulnerable to remote attackers.`", was too obvious for the audience, so feel free to remove that from the suggestion if so!